### PR TITLE
#patch release: fix version command not working when .homeycompose exists but contains no app.json

### PR DESCRIPTION
**Pull request by tjallingt**

### CODE REVIEW

**Review by mrlukasbos — APPROVED** [review]

File: lib/App.js [hunk 1]
```diff
@@ -2263,7 +2273,20 @@ class App {
...
+    const hasComposeFolder = fs.existsSync(path.join(appPath, '.homeycompose'));
+
+    if (
+      hasComposeFolder &&
+      !this.__warnedAboutMissingHomeyComposeManifest &&
+      !fs.existsSync(path.join(appPath, '.homeycompose/app.json'))
+    ) {
+      Log(colors.yellow('⚠ Warning: Could not find a Homey Compose app.json manifest!'));
+      Log(colors.yellow('Using the generated app.json in the root of your app is supported for now,'));
+      Log(colors.yellow('but it is recommended to move your manifest to the .homeycompose/ folder.'));
```
Comment (RobinBol) [inline lib/App.js:2285]: Are these separate log lines needed?

Comment (RobinBol) [inline lib/App.js:2285]:
```suggestion
      Log(colors.yellow('⚠ Warning: Could not find a Homey Compose app.json manifest!'));
      Log(colors.yellow('Using the generated app.json in the root of your app is supported for now,'));
      Log(colors.yellow('but it is recommended to move your manifest to .homeycompose/app.json.'));
```

**Review by RobinBol — APPROVED** [review]

File: lib/App.js (hunk 1 above)
Comment (tjallingt) [reply lib/App.js:2285]: yea alternatively I could break the lines with `+` and end them with `\n` [like this](https://github.com/facebook/react/blob/48740429b4a74e984193e4e2d364d461e4fdc3be/packages/react-reconciler/src/ReactFiberWorkLoop.new.js#L2882-L2897)

Comment (tjallingt) [reply lib/App.js:2285]: can't update in master so https://github.com/athombv/node-homey/pull/167

**Pull request merged**